packages/connector-saml/docs/config-template.json

```diff
@@ -3,11 +3,11 @@
   "idpMetadataXml": "<EntityDescriptor entityID=\"urn:dev.logto.io\" xmlns=\"urn:oasis:names:tc:SAML:2.0:metadata\"><IDPSSODescriptor protocolSupportEnumeration=\"urn:oasis:names:tc:SAML:2.0:protocol\"><KeyDescriptor use=\"signing\">...</IDPSSODescriptor></EntityDescriptor>",
   "signInEndpoint": "<idp-sign-in-endpoint>",
   "x509Certificate": "-----BEGIN CERTIFICATE-----MIIDHTCCAgWg[...]jel7/YMPLKwg+Iau7-----END CERTIFICATE-----",
-  "assertionConsumerServiceUrl": "https://<your-logto-domain>/api/saml-assertion/handler/<saml-connector-id>",
-  "messageSigningOrder": "encrypt-then-sign",
-  "requestSignatureAlgorithm": "RSA_SHA1",
-  "signAuthnRequest": true,
-  "encryptAssertion": true,
+  "assertionConsumerServiceUrl": "https://<your-logto-domain>/api/saml-assertion-handler/<saml-connector-id>",
```

Comment:
Just checked and we need to update the json sample in README.md as well. Please check

Reply:
Updated